radio/src/translations.cpp

```diff
@@ -1,714 +1,725 @@
-/*

```

Comment:
Yeah. There are some line ending problems in the file. The license has only \n while most of the rest of the file has \r\n. Should this file be DOS (\r\n) or Unix (\n) style? Some of the other files have this problem has well.